app/src/main/java/org/gdg/frisbee/android/arrow/ArrowActivity.java

```diff
@@ -313,10 +313,14 @@ public void showNoNfc() {
 
 
     private String getEncryptedMessage() throws Exception {
-        return CryptoUtils.encrypt(Const.ARROW_K, 
```

Comment:
Since this returns null now, it should be null checked. I believe you did it in only 1 place. It is also used in QR generation. 
It would help if you use @Nullable annotation on return statement.